libmattermost.c

```diff
@@ -1155,11 +1168,19 @@ mm_get_users_by_ids_response(MattermostAccount *ma, JsonNode *node, gpointer use
 			if (username != NULL) {          		
 				PurpleBuddy *buddy = purple_buddy_new(ma->account, username, NULL);
 				purple_blist_add_buddy(buddy, NULL, default_group, NULL);
-				purple_blist_node_set_string(PURPLE_BLIST_NODE(buddy), "user_id", mm_user->user_id);
-				purple_blist_node_set_string(PURPLE_BLIST_NODE(buddy), "room_id", mm_user->room_id);
 				g_hash_table_replace(ma->one_to_ones, g_strdup(mm_user->user_id), g_strdup(username));
 				g_hash_table_replace(ma->one_to_ones_rev, g_strdup(username), g_strdup(mm_user->user_id));
+
+				_MM_BLIST_SET(buddy,user,"room_id",mm_user->room_id);

```

Comment:
Don't be scared of whitespace between commas.  We're not being charged by the byte ;)